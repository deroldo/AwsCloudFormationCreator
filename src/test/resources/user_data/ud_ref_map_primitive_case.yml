MyData:
  Template: ref_map_case
  fooMap: bar