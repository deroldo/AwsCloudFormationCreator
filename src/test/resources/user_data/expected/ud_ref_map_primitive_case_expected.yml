---
Resources:
  AnyResource1:
    Type: "AnyAwsType"
    Properties:
      Foo: "bar"
Mappings:
  fooMap1:
    fooMap:
      fooMap:
        Foo: "test"

